types/Game.tsx

```diff
@@ -11,8 +11,8 @@ export interface Game {
     currentQuestion: number; //index of the current question
     questions: Question[]; // An array of resolved Question documents
     startTime: Timestamp;
-    team1: Team; // Resolved Team document
-    team2: Team; // Resolved Team document
+    team1ID: String; // Resolved Team document
```

Comment:
Many parts of the app depend on this being a Team object (makes it easier to pass a full game around without having to make calls to get teams). Either this must be reverted, or the effects of this change need to addressed in other parts of the app (game cards, fetchers, etc)

Reply:
Providing a team object creates a team object within the game rather than a reference to the team. You and Brenna will have to refactor your code to use the getTeam(teamID: String) hook I made to retrieve teams rather than retrieving them directly from the Game.